### ISSUE TITLE
undefined: gobco8

### ISSUE THREAD

**Issue by 503440692** [rillig/gobco#30]
![image](https://github.com/rillig/gobco/assets/66709975/55dea143-cac5-417a-8ec9-a3f1a29d0045)
![image](https://github.com/rillig/gobco/assets/66709975/ace22722-c68c-4087-8aaa-8729aef4d964)
Hi rillig, thank you for developing the tool: gobco. I have encountered some issues while using the tool. Please refer to the two images above for the problem details. It would be appreciated if you could help me resolve this issue.

**Comment by rillig** [comment]
Hi, the code in the screenshot looks quite normal to me. Please send me the whole file, or a reduced version of it that still reproduces the bug. You can run `gobco -keep` to look at the instrumented files, to see where the declaration of `gobco8` is missing.

**Comment by rillig** [comment]
Got it. A minimum reproducer is:
~~~go
switch any(3).(type) {
case uint8:
default:
	switch 1 + 1 {
	case 2:
		break
	}
}
~~~

When instrumenting a switch statement inside a type switch statement, gobco doesn't declare the expression from the inner switch statement. I'm working on a fix.

**Comment by rillig** [comment]
I fixed this bug in [release 1.3.2](https://github.com/rillig/gobco/releases/tag/v1.3.2). Thanks for your report.
